### ISSUE TITLE
Identifiers types not used by UCSB 

### ISSUE THREAD

**Issue by chrissyrissmeyer** [ucsdlib/CA_lib_oration#27]
In UCSD model (but not in UCSB model)
- identifier:collection number (ucsd:collectionNumberId)
- identifier:ark (ucsd:arkId)
- identifier:basket number (ucsd:basketId)
- identifier:call number (modsrdf:callNumber)
- identifier:doi (locid:doi)
- identifier:edm (ucsd:edmId)
- identifier:event number (ucsd:eventId)
- identifier:file name (ucsd:filenameId)
- identifier:igsn (ucsd:igsnId)
- identifier:locus number (ucsd:locusId)
- identifier:negative (ucsd:negativeId)
- identifier:news release number (ucsd:newsReleaseId)
- identifier:oclc number (ucsd:oclcId)
- identifier:registration number (ucsd:registrationId)
- identifier:sample number (ucsd:sampleId)
- identifier:sequence (ucsd:sequenceId)
- identifier:shared shelf (ucsd:sharedShelfId)

**Comment by chrissyrissmeyer** [comment]
RE: identifier:ark (ucsd:arkId)

**Chrissy Rissmeyer**
Aug 31, 2017
greser [1:12 PM] 
Before Hyrax, we used dc:identifier for any identifier that didn't have an approved type and placed the object ARK in ucsd:arkId. Now that it is the object ARK, we no longer need ucsd:arkId. This also means we don't have a generic identifier property.

**Comment by chrissyrissmeyer** [comment]
RE: identifier:doi (locid:doi)

**Matt Critchlow**
Sep 29, 2017
are there still not upstream predicates for DOI, OCLC, IGSN, ISBN, etc? Seems like there would be at least some of them?

**Comment by chrissyrissmeyer** [comment]
UCSB uses dc:identifier for the ARK for the digital resource. It sounds like this is what UCSD may be planning to do? Will need to harmonize since dc:identifer is currently a generic identifier in the model. https://github.com/ucsdlib/CA_lib_oration/blob/c3f467f7ec5acff879d95a34a69852c232682602/object_properties.yml#L1190-L1193

**Comment by chrissyrissmeyer** [comment]
@arwenhutt: Would event ID become redundant with the adoption of an events class?

- identifier:event number (ucsd:eventId)
https://github.com/ucsdlib/CA_lib_oration/blob/c3f467f7ec5acff879d95a34a69852c232682602/object_properties.yml#L1409

**Comment by arwenhutt** [comment]
@chrissyrissmeyer yes, I think it would.  Also, if we add a date property to even (which I think makes sense) date:event as an object property will be redundant too.
